### ISSUE TITLE
Crash when calling doneLoagingAnimation before start animation ends

### ISSUE THREAD

**Issue by rsuper** [leandroBorgesFerreira/LoadingButtonAndroid#3]
When calling `doneLoagingAnimation` after `startAnimation()`, but before the start animation has ended, the application crashes on a nullpointer:
```
Attempt to invoke virtual method 
'void br.com.simplepass.loading_button_lib.animatedDrawables.CircularAnimatedDrawable.stop()' on a null object reference at
br.com.simplepass.loading_button_lib.customViews.CircularProgressButton.doneLoagingAnimation(CircularProgressButton.java:228)
```

As said, it happens in the following situation:
- The button starts animating using `startAnimation()`
- The loading animation is stopped using `doneLoagingAnimation(params)` before the initial start animation has ended (button getting narrower)

In my case I'm running an async task that finishes really quick. Because the button depends on its own start animation, this might be an issue that can arise anytime; especially when using wide buttons on tablet and devices get faster in the future.

Another tiny note: is `doneLoagingAnimation` a typo? (Loaging vs Loading)

**Comment by leandroBorgesFerreira** [comment]
I reproduced the error. I will fix it in some hours. 

Yes, this is a typo. I'll correct it too!

**Comment by leandroBorgesFerreira** [comment]
The problem got corrected in the 1.6.5 version. Please update to this version so you won't face this problem anymore.

**Comment by gunhansancar** [comment]
I'm having the same issue on these lines.
I'm using the current latest version '1.14.0'

Would you mind to add a null check before calling stop in `doneLoadingAnimation(int fillColor, Bitmap bitmap)` method? It seems it is the easiest solution. In my case also the state changes sometimes very quickly and it crashes on those cases with the below logs.
```
    java.lang.NullPointerException: Attempt to invoke virtual method 'void br.com.simplepass.loading_button_lib.animatedDrawables.CircularAnimatedDrawable.stop()' on a null object reference
        at br.com.simplepass.loading_button_lib.customViews.CircularProgressButton.doneLoadingAnimation(CircularProgressButton.java:356)
        at br.com.simplepass.loading_button_lib.customViews.CircularProgressButton$6$1.run(CircularProgressButton.java:570)
```

**Comment by fire1** [comment]
Same here ....

**Comment by leandroBorgesFerreira** [comment]
This problem was corrected prior version 2.0.0. Just tested with a stop and done animation being called before the start animation ends and no problem happens. Fell free to reopen the issue if you still experience this bug.
